grouper/fe/templates/permission.html

```diff
@@ -58,56 +58,30 @@
             <table class="table table-elist">
                 <thead>
                     <tr>
-                        <th class="col-sm-2">Group</th>
-                        <th class="col-sm-5">Argument</th>
+                        <th class="col-sm-2">Permission Grants</th>
+                        <th class="col-sm-5">Type</th>
```

Comment:
Don't think this second column is necessary at all, given that the first column already specifies the type